scripts/container-postCreate.sh

```diff
@@ -9,5 +9,6 @@ pip install -r requirements.txt
 git config --global --add safe.directory $(pwd)
 pre-commit install --hook-type pre-commit
 pre-commit install --hook-type pre-push
+mypy --install-types --non-interactive
```

Comment:
Sorry super basic question - can you remind me how do we use this bash script?

Reply:
devcontainer runs this after it builds the container.